### DISCUSSION TITLE
Ortaklık programı ekleyin

### DISCUSSION THREAD

**Discussion by Ruhulbeyan** [IBAX-io/go-ibax#474] [Ecolibs]
### Clear and concise description of the problem

Size daha çok  faydası olacak olan ortaklık programı ekleyin

### Suggested solution

Ortaklık programı ekleyin

### Alternative

Yatırım yapabileceğimiz bir şirket ortaklık platformu istiyoruz

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment]
We are in the process of making a eco-incentive programs, including affiliate programs, and you can also contact @IBAXKyle directly via Telegram
